src/main/resources/assets/chisel/lang/ru_RU.lang

```diff
@@ -1,13 +1,13 @@
 item.chisel.chisel_iron.name=Стамеска
 item.chisel.chisel_diamond.name=Алмазная стамеска
 item.chisel.chisel_hitech.name=Высокотехнологичная стамеска
-item.chisel.offsettool.name=Ender Offset Wand
+item.chisel.offsettool.name=Эндер-жезл смещения
 
 item.chisel.chisel.desc.gui=ПКМ, чтобы открыть интерфейс
 item.chisel.chisel.desc.lc1=ЛКМ, чтобы высечь блоки в мире
```

Comment:
One other thing, I've done a strikethrough on the value for Lang key `item.chisel.chisel.desc.lc1` since Left-click chiseling is not yet implemented. I don't see one here, feel free to fix that. Otherwise I can pull but people will not know otherwise.